### PR TITLE
Mac: Known issue - Fullscreen must use all monitors

### PR DESCRIPTION
**Pull request by adventurist**
- Adding a known issue
- Removing "Will not fix" from previous known issue requiring Mission Control to be set to use separate spaces for each monitor

### CODE REVIEW

File: mac/README.md [hunk 1]
```diff
@@ -201,4 +201,5 @@ Here are the steps to configure Mission Control correctly for FULLSCREEN mode:
 ## 7. Known Issues / Limitations
 ### Known Issues
 - No local and remote video seen on video call when the vdi mac recovers from "sleep"  action after 4 minutes. `KAJ-1127`
-- Fullscreen mode with multiple monitors only functions correctly when Mission Control's *Displays have separate Spaces* is set to the default *checked* setting `Will not fix`
+- Fullscreen mode with multiple monitors only functions correctly when Mission Control's *Displays have separate Spaces* is set to the default *checked* setting
+- Fullscreen mode only functions correctly when Citrix is configured to use ALL monitors. `KAJ-1314`
```
Comment (brenthompson) [inline mac/README.md:205]: I'd reword these to something less black and white, like "for proper operation...." 
"for proper operation Citrix should be configured to use all monitors"
words like "should", "may", "may not", etc are better for customer-facing docs than "only functions correctly"

**Review by brenthompson — CHANGES_REQUESTED** [review]

File: mac/README.md (hunk 1 above)
Comment (adventurist) [reply mac/README.md]: I've amended it to be a bit more customer-friendly

File: mac/README.md [hunk 2]
```diff
@@ -191,14 +191,15 @@ Congratulations! You have built, signed, installed and configured the KandyDista
...
+Here are the steps to optimize Misson Control for FULLSCREEN mode:
 1. Open System Preferences -> Mission Control
 2. Ensure the checkbox next to "`Displays have separate Spaces`" is **CHECKED**/**ACTIVE**
 
 
 ## 7. Known Issues / Limitations
 ### Known Issues
 - No local and remote video seen on video call when the vdi mac recovers from "sleep"  action after 4 minutes. `KAJ-1127`
-- Fullscreen mode with multiple monitors only functions correctly when Mission Control's *Displays have separate Spaces* is set to the default *checked* setting `Will not fix`
+- Fullscreen mode with multiple monitors functions best when Mission Control's *Displays have separate Spaces* is set to the default *checked* setting
```
Comment (brenthompson) [inline mac/README.md:204]: not to be too picky, but should we mention that this is in reference to remote app window? cause right now it's not clear what around Fullscreen mode doesn't work correctly

**Review by brenthompson — APPROVED** [review]
LGTM
